.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     'plugin:import/typescript',
     'raine'
   ],
+  ignorePatterns: ['build'],
```

Comment:
I'd swear that I thought by default ESLint respects .gitignore but apparently it doesn't.

Personally I just use `--ignore-path .gitignore` so that I don't have 2 places to handle these, but either way your call :)

Reply:
It uses the same format as .gitignore but it doesn't read from it.

Ah, I see. That's smart.